src/rpcblockchain.cpp

```diff
@@ -33,9 +33,24 @@ double GetDifficulty(const CBlockIndex* blockindex)
             blockindex = chainActive.Tip();
```

Comment:
Independent of the PR, but can ``chainActive.Tip()`` have changed between these two calls? I'd prefer this which is more clearly locally correct:
```
if (blockindex == NULL) {
    blockindex = chainActive.Tip();
    if (blockindex == NULL)
        return 1.0;
}
```